services/email.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+     host: '' || process.env.SAYONIKA_EMAIL_HOST,
```

Comment:
Set some sane defaults, see python code for examples

Reply:
We're following the Env var conventions here